vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ReferenceBlock.java

```diff
@@ -0,0 +1,30 @@
+package edu.asu.diging.vspace.core.model.impl;
```

Comment:
this is not a model class, is it? It's only used in the controllers? It shouldn't be in the model package.